frontend/src/react/Exercice/Exercice.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import ExerciceDisplayer from './ExerciceDisplayer';
 
-export default class Exercice extends React.PureComponent {
+export default class Exercice extends React.Component {
 
   state = {
-    question: [1]
+    question:[
+      {"questionTitle": '', "questionScale": null, "questionContent": ''},
```

Comment:
The backend is going to send different names of attributes; it will be more like : 
`{title: '', scale: 1, content: ''}`

Reply:
True enough, but then we can't identify if it's a question or an exercice title, mapping will be needed here I think

Reply:
The component itself knows it, doesn't it ?
Anyway, this will certainly be refactored when linking with the backend I guess

Reply:
Yes but the higher component which holds the logic need a name in the store/state, it can't be the same name otherwise there will override each other.

Agree with you on this point !